package.json

```diff
@@ -43,4 +43,4 @@
     "torrust-index-types-lib": "^0.2.0",
     "uuid": "^9.0.0"
   }
-}
```

Comment:
no need to edit this file (removing the trailing newline)

Reply:
Fixed with new commit.

Reply:
Hi @mario-nt, in general, I prefer to rewrite the PR commits instead of revert changes if they are not merged yet. Just to have the history as clean and straightforward as possible.

Reply:
Hi @josecelano, I thought about that too and wasn't sure about wich approach to take. I will go and modify the commits as I think too, it creates a clearer commit history.